Backend/app/rider-platform/rider-app/Main/src/Domain/Action/Beckn/OnUpdate.hs

```diff
@@ -400,11 +400,12 @@ onUpdate = \case
         else return False
     let newQuote = quote{id = Id quoteId_, createdAt = now, updatedAt = now}
         newBooking = booking{id = bookingId, quoteId = Just (Id quoteId_), status = SRB.CONFIRMED, isScheduled = newIsScheduled, bppBookingId = Just newBppBookingId, startTime = max now booking.startTime, createdAt = now, updatedAt = now}
+        flowStatus = if newIsScheduled then DPFS.IDLE else DPFS.WAITING_FOR_DRIVER_ASSIGNMENT {bookingId = bookingId, validTill = searchReq.validTill, fareProductType = Just $ STB.getFareProductType booking.bookingDetails, tripCategory = booking.tripCategory}
```

Comment:
Why this change?

Reply:
when driver accepts booking lets say before 2 hours then he cancels it then the screeen was going to reallocation screen but we dont want that screen . so here newIsScheduled will be true in that case if cancelled before more than 30 minutes and in case of false will show reallocation screen